pkg/controller/infrastructure/actuator.go

```diff
@@ -46,9 +48,12 @@ type actuator struct {
 }
 
 // NewActuator creates a new Actuator that updates the status of the handled Infrastructure resources.
-func NewActuator(disableProjectedTokenMount bool) infrastructure.Actuator {
+func NewActuator(mgr manager.Manager, disableProjectedTokenMount bool) infrastructure.Actuator {
 	return &actuator{
 		disableProjectedTokenMount: disableProjectedTokenMount,
+		decoder:                    serializer.NewCodecFactory(mgr.GetScheme()).UniversalDecoder(),
```

Comment:
Should we not use `serializer.EnableStrict`?

Reply:
Added to [bd8722e](https://github.com/gardener/gardener-extension-provider-openstack/pull/655/commits/bd8722e6d181b6a50989fa31c026ed34f67bbe85)